.ci/UbuntuBionic/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
```

Comment:
Looks like you moved the Dockerfile to a subdirectory, but the old file in `.ci/` has not been deleted.

Reply:
strange, I guess `git mv` doesn't actually remove the old file?